compiler/AST/expr.cpp

```diff
@@ -1139,7 +1140,7 @@ Type* getRefTypesForWideThing(GenRet wide, Type** wideRefTypeOut)
         ret = wide.chplType->getField("addr")->typeInfo();
         wideRefType = wide.chplType;
       } else {
-        INT_ASSERT(0);
+        //INT_ASSERT(0);
```

Comment:
Without this assert, getRefTypesForWideThing returns NULL. But the assert was there to indicate 'wide' was not in fact a wide thing. So it would be better to just not call it at all in that case, and leave the assert in, I think.